.github/dependabot.yml

```
@@ -0,0 +1,14 @@
+# Basic set up for three package managers
+
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
```

I would also be okay with getting npm monthly to reduce the noise. 

```suggestion
      interval: 'monthly'
```